docs/docs/running-hello-world.md

```diff
@@ -73,216 +75,235 @@ The web server loads the enclave and waits for requests from the client. Amongst
-enclave that has been loaded. The private key that corresponds to the "Session encryption key" (printed above) is only
-available inside the enclave. Our client will use the "Session encryption key" to encrypt data to send into the enclave.
+This output is the [remote attestation](enclaves.md#remote-attestation), an object which proves certain information
+about the enclave. The private key corresponding to the "Session encryption key" (printed above) is only available
+inside the enclave. The client will use the "Session encryption key" to encrypt data to send to the enclave.
+
+After the server starts up, it will be ready to communicate with the client on http://localhost:8080.
 
-Once the server is done starting up it will be ready to communicate with the client on http://localhost:8080.
-You can proceed to [Run the client](#run-the-client) when you see the following output:
+2. Proceed to [Run the client](#run-the-client) when you see the following output:
 
 ```text
-[main] INFO com.r3.conclave.host.web.EnclaveWebHost$Companion - Started EnclaveWebHost.Companion in <SECONDS> seconds 
+[main] c.r.c.host.web.EnclaveWebHost$Companion  : Started EnclaveWebHost.Companion in <SECONDS> seconds 
 ```
 
-!!! warning
-    The sample uses Conclave mail which handles encryption and authentication, but the web host protocol makes use of
-    request headers which are not encrypted. Because of this, it is still important to use HTTPS for anything other than
-    internal development. Setting up an HTTPS connection is beyond the scope of this tutorial, but you may wish to look
-    at configuring Spring Boot or setting up a reverse proxy such as Nginx or Apache. See
-    [Conclave web host](conclave-web-host.md) for more information.
+!!!Warning
+
+    The sample uses Conclave Mail which handles encryption and authentication. But the web host protocol uses
+    unencrypted request headers. So, it is important to use HTTPS for anything other than internal development.
+    Setting up an HTTPS connection is beyond the scope of this tutorial. You can try configuring Spring Boot or 
+    setting up a reverse proxy such as Nginx or Apache. See [Conclave web host](conclave-web-host.md) for more 
+    information.
 
 ## Run the client
 
-Run the client with
+After running the host:
+
+1. Open another command line interface.
+2. Navigate to the hello-world sample.
 ```bash
-java -jar client/build/libs/client.jar \
-  # A constraint which defines which enclaves the client will accept.
-  "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1 SEC:INSECURE" \
-  # The string to reverse
-  reverse-me
+cd conclave-tutorials/hello-world
 ```
+3. Run the client with the command:
+```bash
+java -jar client/build/libs/client.jar "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1
+SEC:INSECURE" reverse-me
+```
+In the above command, the parameters in quotes are the constraints, and 'reverse-me' is the string to be reversed.
```

Comment:
We lost comment about constraint here, so I would add a link on the word constraints to the next section which explains it a bit more.

Reply:
Done.